seed/lib/superperms/orgs/migrations/0004_organization_measurement_system.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.13 on 2017-08-15 16:22
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('orgs', '0003_auto_20160412_1123'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='organization',
+            name='measurement_system',
+            field=models.IntegerField(choices=[(1, b'United States customary units'), (2, b'Metric (SI) units')], default=1),
```

Comment:
I think we want the USC to be termed IP Units.

Reply:
Dealt with in a new migration.

Generally, is migration history OK or do you prefer it if I just alter the existing migration?